app/controllers/claims_controller.rb

```diff
@@ -96,7 +96,9 @@ def redirect_to_existing_claim_journey
   end
 
   def set_backlink_path
-    @backlink_path = claim_path(current_journey_routing_name, previous_slug) if previous_slug.present?
+    if previous_slug.present? && Journeys::PageSequence::DEAD_END_SLUGS.exclude?(current_slug)
```

Comment:
We do this as we don't want to show backlinks on the ineligible pages on the LUP only journey.
Backlinks only render if `@backlink_path` has been set.

We should consider moving `DEAD_END_SLUGS` into the `SlugSequence` once everything is moved over to use the navigator.